qrcode/image/base.py

```diff
@@ -12,13 +12,13 @@ def __init__(self, border, width, box_size, *args, **kwargs):
         self.pixel_size = (self.width + self.border*2) * self.box_size
         self._img = self.new_image(**kwargs)
 
-    def drawrect(self, row, col):
+    def drawrect(self, row, col):  # pragma: no cover
```

Comment:
Let's make this an `@abc.abstractmethod` instead -- we don't need the comment, and it doesn't need a return value or `raise` line